app/models/manageiq/providers/openstack/inventory/persister/definitions/cloud_collections.rb

```diff
@@ -119,6 +119,7 @@ def add_orchestration_stacks(extra_properties = {})
   def add_orchestration_templates
     add_collection(cloud, :orchestration_templates) do |builder|
       builder.add_properties(:model_class => ::OrchestrationTemplate)
+      builder.add_default_values(:ems_id => manager.id)
```

Comment:
hm, now I am not sure if adding it to all templates can cause problems

Reply:
maybe it will be better to add it to parser

Reply:
Any further thoughts on this?

Reply:
I am not sure :-)